src/components/EntityInfo.vue

```diff
@@ -223,42 +260,32 @@ export default class EntityInfo extends Mixins(EnumMixin) {
 
 <!-- eslint-disable max-len -->
 <style lang="scss" scoped>
-// FUTURE: Explore how to expose this globally without having to include in each module
-// see https://cli.vuejs.org/guide/css.html#automatic-imports
-// see https://vue-loader-v14.vuejs.org/en/features/postcss.html
 @import '@/assets/styles/theme.scss';
 
 #entity-info {
   background: $BCgovInputBG;
+}
 
-  .breadcrumb {
-    padding: 0;
-  }
+.v-breadcrumbs li {
+  font-size: 0.75rem;
+}
 
-  .v-breadcrumbs li {
-    font-size: .75rem;
+::v-deep {
+  .v-breadcrumbs a {
+    color: $gray8 !important;
```

Comment:
NB: this is needed to increase the CSS selector specificity (which was previously done by nesting this inside `#entity-info` block (but large-scale nesting should be avoided).